bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -487,8 +512,6 @@ spec:
                         value: quay.io/metallb/speaker:main
                       - name: CONTROLLER_IMAGE
                         value: quay.io/metallb/controller:main
-                      - name: ENABLE_WEBHOOK
```

Comment:
This belongs to another commit maybe?

Reply:
which commit? i can split it if you think it's needed

Reply:
This commit is about bumping metallb version, while here we also remove the env variable (and we are removing the ENABLE_WEBHOOK management in subsequent commits iirc).
Not sure it makes sense to split it, maybe (but I won't hold against this), it's goes better together with one of the  ENABLE_WEBHOOK commits

Reply:
ok